packages/frontend/editor-ui/src/composables/usePushConnection/handlers/executionFinished.ts

```diff
@@ -467,13 +468,12 @@ export function setRunExecutionData(
 
 	workflowsStore.executingNode.length = 0;
 
-	if (normalize) {
-		// As a temporary workaround for https://linear.app/n8n/issue/PAY-2762,
-		// remove runs that is still 'running' status when execution is finished
-		removeRunningTaskData(execution as IExecutionResponse);
-	}
-
-	workflowsStore.setWorkflowExecutionData(workflowExecution as IExecutionResponse);
+	workflowsStore.setWorkflowExecutionData({
+		...workflowExecution,
+		status: execution.status,
+		id: execution.id,
+		stoppedAt: execution.stoppedAt,
```

Comment:
The logs view depend on these fields to show 'Success in Xms' after the execution is finished.